sdcm/nemesis.py

```diff
@@ -2692,6 +2710,16 @@ def disrupt(self):
         self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
 
 
+class NemesisSequence(Nemesis):
```

Comment:
this one should go to the `COMPLEX_NEMESIS` list below

Reply:
I rather keeping it as regular nemesis.